<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>jboss on Miro&#39;s World</title>
    <link>https://miroadamy.com/tags/jboss/</link>
    <description>Recent content in jboss on Miro&#39;s World</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <lastBuildDate>Sat, 11 Apr 2015 11:22:48 +0800</lastBuildDate><atom:link href="https://miroadamy.com/tags/jboss/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Fixing the shutdown problem with JBoss</title>
      <link>https://miroadamy.com/posts/2015-04-11-jboss-shutdown/</link>
      <pubDate>Sat, 11 Apr 2015 11:22:48 +0800</pubDate>
      
      <guid>https://miroadamy.com/posts/2015-04-11-jboss-shutdown/</guid>
      <description>Observed at QA environment. During automated shutdown sequence, the exception was thrown:
$ bin/gdi-qa-com1.sh stop JBOSS_CMD_STOP = java -classpath /opt/jboss/jboss-as/bin/shutdown.jar:/opt/jboss/jboss-as/client/jnet.jar org.jboss.Shutdown --shutdown --user=admin --password=admin -s jnp://0.0.0.0:1099 Exception in thread &amp;#34;main&amp;#34; javax.naming.CommunicationException: Could not obtain connection to any of these urls: 0.0.0.0:1099 [Root exception is javax.naming.CommunicationException: Failed to connect to server /0.0.0.0:1099 [Root exception is javax.naming.ServiceUnavailableException: Failed to connect to server /0.0.0.0:1099 [Root exception is java.net.ConnectException: Connection refused]]] at org.jnp.interfaces.NamingContext.checkRef(NamingContext.java:1763) at org.</description>
    </item>
    
    <item>
      <title>ATG &#43; JBoss rant</title>
      <link>https://miroadamy.com/posts/2014-09-03-atg-rant/</link>
      <pubDate>Wed, 03 Sep 2014 11:22:48 +0800</pubDate>
      
      <guid>https://miroadamy.com/posts/2014-09-03-atg-rant/</guid>
      <description>Symptoms restarting after kill -9 of CSC instance does not succeed. The server never starts, the WAR files are not deployed properly, socket 8080 is open but nobody answers (not even with wget on localhost).
The exception: &amp;#34;main&amp;#34; prio=10 tid=0x0000000051a39000 nid=0x30fb in Object.wait() [0x00002b5df9d6c000] java.lang.Thread.State: TIMED_WAITING (on object monitor) at java.lang.Object.wait(Native Method) - waiting on &amp;lt;0x00000007e36ae9d0&amp;gt; (a atg.service.lockmanager.ClientLockEntry) at atg.service.lockmanager.ClientLockEntry.acquireWriteLock(ClientLockEntry.java:399) - locked &amp;lt;0x00000007e36ae9d0&amp;gt; (a atg.service.lockmanager.ClientLockEntry) at atg.service.lockmanager.ClientLockManager.acquireWriteLock(ClientLockManager.java:1137) at atg.repository.search.indexing.ConfigStatePersister.acquireLock(ConfigStatePersister.java:1844) at atg.</description>
    </item>
    
  </channel>
</rss>
